cpp/velox/benchmarks/data/plan/select_decimal.json

```diff
@@ -0,0 +1,61 @@
+{
+    "relations": [
```

Comment:
We may upgrade substrait, it is not recommended to save plan json file here, please save it in your local or generate as https://github.com/oap-project/gluten/blob/main/docs/developers/MicroBenchmarks.md#try-the-example

Reply:
ok

Reply:
because don't add the plan `json` file can't pass the ci, so I keep this file, I will merge select_decimal.json into select.json in the next orc related PR.

Reply:
Talked with @zuochunwei offline, the two json files are considered to be merged into one next. To replace, Scala unit test can also be used to test file scan.